Campus-iOS/LoginComponent/Views/TokenConfirmationView.swift

```diff
@@ -91,34 +110,119 @@ struct TokenConfirmationView: View {
+                                            tokenState = .notChecked
+                                            buttonBackgroundColor = .tumBlue
+                                        }
+                                    }
+                                }
+                            case .active:
+                                HStack {
+                                    Image(systemName: "checkmark.circle.fill")
+                                    Text("Token active").lineLimit(1).font(.body)
+                                }
+                                .padding()
+                                .onAppear() {
+                                    Task {
+                                        try? await Task.sleep(nanoseconds: 3_000_000_000)
+                                        withAnimation() {
+                                            tokenPermissionButton = true
+//                                            showCheckTokenButton = false
+                                        }
+                                    }
+                                }
+                            }
+                        }
                         .frame(width: 200, height: 48, alignment: .center)
+                        .font(.title)
+                        .foregroundColor(.white)
+                        .background(buttonBackgroundColor)
+                        .cornerRadius(10)
+                        .buttonStyle(.plain)
+                    }
+                    
+                    if tokenPermissionButton, let model = self.viewModel.model  {
+                        NavigationLink(destination: TokenPermissionsView(viewModel: TokenPermissionsViewModel(model: model)).navigationTitle("Check Permissions")) {
+                            Text("Next")
+                                .lineLimit(1)
+                                .font(.body)
+                                .frame(width: 200, height: 48, alignment: .center)
+                                .foregroundColor(.white)
+                                .background(.green)
+                                .cornerRadius(10)
+                                .buttonStyle(.plain)
+                        }
+                    }
+                    
+                    if showTokenHelp {
+                        Spacer()
+                        Text("Please activate the latest token on TUMOnline").foregroundColor(.red)
+                        Spacer()
+                    }
+                    
+                    Spacer()
+                    
+                    let mailToString = "mailto:app@tum.de?subject=[IOS - Token]&body=Hello, I have an issue activating the token of Campus Online in the TCA version \(Bundle.main.appVersionShort) on \(ProcessInfo().operatingSystemVersion.fullVersion). Please describe the problem in more detail.".addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed)
```

Comment:
Maybe add the token permissions to this mail?

Reply:
So that in the support we can directly check if the token has the respective rights?

Reply:
You mean that we get the info, if the token is already active or not?
The permissions are handled on the next view and thus cannot have any impact to be allowed to get to the next step. On this view you must just activate the token, not the permissons. This is of the next view. :)

Reply:
✅

Reply:
Ah okay, misplaced that :D

Reply:
But for support requests after activating the token, can u pass the respective rights of the token with the support mail?